sources/packages/backend/apps/queue-consumers/src/services/application/application.service.ts

```diff
@@ -103,7 +106,7 @@ export class ApplicationService {
             submittedStatus: StudentAssessmentStatus.Submitted,
           },
         )
-        .andWhere(`NOT EXISTS (${inProgressStatusesExistsQuery})`)
+        .andWhere(`NOT EXISTS (${this.inProgressStatusesExistsQuery})`)
         .setParameter("inProgressStatuses", [
           StudentAssessmentStatus.Queued,
           StudentAssessmentStatus.InProgress,
```

Comment:
Even considering that cancellation will be requested for now only for the original assessment I still would like to advise to include here the `Cancellation requested` and `Cancellation queued`.